src/data/sample-data.js

```diff
@@ -192,6 +192,14 @@ export const sampleUsersData = [
     "name": "Ephraim Goyette",
     "id": 2,
     "pantry": [
+      {
+        "ingredient": 1009016,
+        "amount": 6
+      },
+      {
+          "ingredient": 1002046,
+          "amount": 2
+      },
```

Comment:
Thank you, Lauren!!

Reply:
Thank you for refactoring the data for the pantry class!